smoketest_lint/cmd/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/greenbone/openvas-scanner/smoketest_lint/test"
+)
+
+func main() {
+
+	openvasDir := flag.String("openvasDir", "", "location of the openvas-nasl-lint executable, has to be absolute or relative to test files directory")
```

Comment:
Instead of just adding the root dir of a executable I would rather use executable or e and set the default to openvas-nasl-lint?

Reply:
Also a little bit of output when running would be nice.